### PR TITLE
[QNN] Support 4D padding.

### PR DESCRIPTION
**Pull request by anijain2305**
As Title

@FrozenGene 

https://github.com/apache/incubator-tvm/pull/4803

### CODE REVIEW

File: python/tvm/relay/qnn/op/qnn.py [hunk 1]
```diff
@@ -280,6 +281,9 @@ def conv2d(data,
         The computed result.
     """
 
+    # TODO enforce 4-way padding in topi/nn/conv2d after #4644 merged
+    # convert 2-way padding to 4-way padding
```
Comment (FrozenGene) [inline python/tvm/relay/qnn/op/qnn.py]: Why this is related with #4644?

Comment (anijain2305) [reply python/tvm/relay/qnn/op/qnn.py]: This is to make Relay Conv2d and QNN conv2d handle 4D padding in same manner. Relay Conv2d looks like this

https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/op/nn/nn.py#L204-L206

I guess QNN changes got missed earlier.

Comment (FrozenGene) [reply python/tvm/relay/qnn/op/qnn.py]: Ok. Got it.

**Comment by FrozenGene** [comment]
Thanks @anijain2305 Merged.

**Pull request merged**